### PR TITLE
Issue 4 delete an entry

### PR DESCRIPTION
**Pull request by bryanvaldovinos**
https://user-images.githubusercontent.com/103273265/172952528-919f48c7-9289-45a3-93fc-70dbd2c61848.mov

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -112,3 +112,64 @@ textarea {
...
+
+.modal-content {
+  background-color: white;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  text-align: center;
+  align-items: center;
+  min-height: 20vh;
+  margin: 250px 525px;
```
Comment (codingcodymiller) [inline css/styles.css]: Why this huge margin? There is likely a better way to do this.

File: css/styles.css [hunk 2]
```diff
@@ -112,3 +112,64 @@ textarea {
...
+  background-color: #b6b6b6;
+  font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
+  font-size: 1rem;
+  border: none;
+  border-radius: 5px;
+  float: left;
+  margin-right: 100px;
+}
+
+.modal-button-2 {
```
Comment (codingcodymiller) [inline css/styles.css]: Numbered class name? 😱

File: css/styles.css [hunk 3]
```diff
@@ -112,3 +112,64 @@ textarea {
...
+  color: white;
+  background-color: red;
+  font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
+  font-size: 1rem;
+  border: none;
+  border-radius: 5px;
+  float: right;
+}
+
+.space {
```
Comment (codingcodymiller) [inline css/styles.css]: space what?

File: js/main.js [hunk 4]
```diff
@@ -32,7 +32,7 @@ function submit(event) {
     data.editing.title = contact.elements.title.value;
     for (var i = 0; i < data.entries.length; i++) {
       if (data.entries[i].EntryId === data.editing.EntryId) {
-        uList.replaceWith(entryTree(data.editing));
+        uList.getElementsByTagName('li')[i].replaceWith(entryTree(data.editing));
```
Comment (codingcodymiller) [inline js/main.js]: You should stick to `querySelector`

File: js/main.js [hunk 5]
```diff
@@ -138,12 +154,31 @@ function updateEntry(evento) {
...
 
+function confirmDelete() {
+  view.className = 'hidden';
+  viewTwo.className = '';
+  modalShow.className = 'modal hidden';
+  contact.reset();
+  img.setAttribute('src', 'images/placeholder-image-square.jpg');
+  for (var i = 0; i < data.entries.length; i++) {
+    if (data.entries[i].EntryId === data.editing.EntryId) {
+      uList.getElementsByTagName('li')[i].remove();
```
Comment (codingcodymiller) [inline js/main.js]: You should stick to `querySelector`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by bryanvaldovinos** [comment]
https://user-images.githubusercontent.com/103273265/172970308-c970926e-0655-4e0d-93ee-8cea73a70748.mov

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**